### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by kerlos-bahaa**
-Add desktop first page
-Add desktop work section
-Add desktop about me section
-Add desktop contact me section

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <title>Portfolio</title>
     <!-- Css fie  -->
     <link rel="stylesheet" href="Css/master.css" />
+    <link rel="stylesheet" href="Css/Deskto.css" media="(min-width: 768px) ">
```
Comment (tufoinnkuo10) [inline Index.html:10]: - [ ] Great job setting the media query from here. Please remember the guidelines are to test and prepare you on some particular aspect, and on this you were to use media queries for the Desktop breakpoint in the CSS file. This was to see how you can differentiate the mobile and desktop in one CSS file. Please, you can use the other method later on. It was clearly mentioned in the project requirement for you as well 👇🏾 


Screenshot (917)_LI

File: Index.html [hunk 2]
```diff
@@ -26,244 +27,263 @@
...
+  
+  <!-- third page -->
+  
+  <div class="repeated-div">
+    <img
+      class="repeated-img"
+      src="./picture/Snapshoot Portfolio3.png"
+      alt=""
+    />
+    <img class="pcrepeated-img" src="./picture/pc3.png" alt="">
```
Comment (tufoinnkuo10) [inline Index.html:161]: - [ ] please add a little padding left and right to the desktop cards

File: Index.html [hunk 3]
```diff
@@ -26,244 +27,263 @@
...
-          <li>
-            <a href="#"> <img src="./picture/Vector.png" alt="" /></a>
-          </li>
-          <li>
-            <a class="github" href="https://github.com/kerlos-bahaa">
-              <i class="fa-brands fa-github"></i>
-            </a>
-          </li>
+  <div class="aboutmyself-top">
+    <h2 class="About-name">About Myself</h2>
```
Comment (tufoinnkuo10) [inline Index.html:221]: - [ ] please kindly add a little padding left to the about section so it matches the original design

Screenshot (916)_LI

File: Index.html [hunk 4]
```diff
@@ -26,244 +27,263 @@
...
     </section>
-
     <footer>
       <section class="footer-section">
         <form
           action="https://formspree.io/f/xayzpwaq"
           method="post"
           target="_blank"
         >
+        <!-- contact me  -->
```
Comment (tufoinnkuo10) [inline Index.html:286]: - [ ] please kindly use the correct background image in the desktop contact section like in the original design.
yours
Screenshot (913)
figma
Screenshot (914)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @kerlos-bahaa 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Responsive

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**